hoplite-core/src/main/kotlin/com/sksamuel/hoplite/ConfigLoaderBuilder.kt

```diff
@@ -310,7 +310,7 @@ class ConfigLoaderBuilder private constructor() {
   /**
    * When enabled, allows placeholder substitutions like ${foo} not to cause an error if they are not resolvable.
    */
-  @Deprecated("Use SubstitutionMode")
+  @Deprecated("Use ContextResolverMode", ReplaceWith("withContextResolverMode(ContextResolverMode.SkipUnresolved)"))
```

Comment:
Hmm. As a side note, one of [our tests fails](https://github.com/oss-review-toolkit/ort/actions/runs/10890826383/job/30220500188?pr=9156#step:4:2859) with

> java.lang.IllegalArgumentException: Failed to load ORT configuration: Unresolved substitution [https://my-repo.example.org(?](https://my-repo.example.org(/?)<repoPath>.*) -> ssh://my-mapped-repo.example.org${repoPath} at (src/main/resources/reference.yml:275:29)

after applying this replacement. Is this not the new equivalent?

Reply:
I've added a commit that should fix the issue, @sksamuel.